src/main/io/osd.h

```diff
@@ -28,11 +28,11 @@
 
 // Character coordinate
 
-#define OSD_POSITION_BITS 5 // 5 bits gives a range 0-31
-#define OSD_POSITION_XY_MASK ((1 << OSD_POSITION_BITS) - 1)
-#define OSD_POS(x,y)  ((x & OSD_POSITION_XY_MASK) | ((y & OSD_POSITION_XY_MASK) << OSD_POSITION_BITS))
-#define OSD_X(x)      (x & OSD_POSITION_XY_MASK)
-#define OSD_Y(x)      ((x >> OSD_POSITION_BITS) & OSD_POSITION_XY_MASK)
+//#define OSD_POSITION_BITS 5 // 5 bits gives a range 0-31
```

Comment:
This abstraction is still used when sending OSD position over MSP. But it may be moved into some MSP-related file